shared/control/parallel_dd.control

```diff
@@ -10,12 +10,16 @@ the files system.
 """
 
 # YOU NEED TO SPECIFY A FILESYSTEM
-if os.path.exists("/dev/sdb"):
-    fs = job.filesystem('/dev/sdb', job.tmpdir)
-elif os.path.exists("/dev/hdb"):
-    fs = job.filesystem("/dev/hdb", job.tmpdir)
-else:
-    fs = job.filesystem("/dev/vdb", job.tmpdir)
+path = glob.glob("/dev/[hsv]d?")
+path.sort()
+fs = path[-1]
+platform = subprocess.check_output("uname -m", shell=True)
+if platform not in ("x86_64", "i386"):
+    if not subprocess.check_output("lsscsi"):
```

Comment:
if use virtio_blk, the system disk will be /dev/vdb ,howerver the disk will be /dev/vda. when use virtio_scsi ,the system_disk will still be /dev/vda.